### PR TITLE
GDPR consent, localization of CESNET template

### PR DESCRIPTION
**Pull request by dBucik**
Design of consent page has been changed, it should now be OK with GDPR and exact match to the templates we are using for other consents. Structure of the tags has been refactored. The CESNET template now has support for language, texts are read from properties files.

### CODE REVIEW

File: oidc-idp/src/main/java/cz/muni/ics/oidc/PerunOidcConfig.java [hunk 1]
```diff
@@ -58,6 +59,14 @@ public String getTheme() {
         return theme;
     }
 
+    public void setGroupCheckEnabled(Boolean groupCheckEnabled) {
```
Comment (martin-kuba) [inline oidc-idp/src/main/java/cz/muni/ics/oidc/PerunOidcConfig.java]: Group check should not be a general setting for the whole OIDC server, it should be per-client setting.

Comment (dBucik) [reply oidc-idp/src/main/java/cz/muni/ics/oidc/PerunOidcConfig.java]: Should I therefore move this configuration option to a new object? I would move the "theme" config there as well. By default the theme would be set to original provided by MITREid and group checking would be disabled.

Comment (martin-kuba) [reply oidc-idp/src/main/java/cz/muni/ics/oidc/PerunOidcConfig.java]: I do not see a reason for moving this configuration to a new object. My complaint was that this setting should be enabled per client, not per server.

**Comment by dBucik** [comment]
The feature to disable the group check has been dropped, if we want to do it in future we will probably connect it to the Perun which will store if the membership should be checked or not.

**Pull request closed without merging**